source/css/_db-ui-core.variables.scss

```diff
@@ -1,10 +1,18 @@
+/* *****
+	This file summarizes the general variables relevant for using DB UI Core with SCSS.
+***** */
+
 // Define the path for both image and font folders
 $icons-path: "../../icons/" !default;
 $images-path: "../../images/" !default;
 $fonts-path: "../../fonts/" !default;
 
 @import "@db-ui/base/build/scss/db-ui-base";
 
+:root {
+	@extend %db-ui-functional;
```

Comment:
why do we use functional here as default?

Reply:
I think that the developer documentation is a functional app :P